src/Compilers/CSharp/Portable/Symbols/Source/SourceAssemblySymbol.cs

```diff
@@ -135,7 +135,8 @@ internal sealed partial class SourceAssemblySymbol : MetadataOrSourceAssemblySym
 
             if (!compilation.Options.CryptoPublicKey.IsEmpty)
             {
-                _lazyStrongNameKeys = StrongNameKeys.Create(compilation.Options.CryptoPublicKey, MessageProvider.Instance);
+                // FEATURE: private key should likely not be null.
```

Comment:
>  // FEATURE: p [](start = 14, length = 15)

Use PROTOTYPE instead of FEATURE. This is the way we track all TODOS in feature branche.s